packages/react/src/themes/teams/components/Tooltip/tooltipPointerSvgUrl.ts

```diff
@@ -0,0 +1,4 @@
+export default (backgroundColor: string) =>
+  `url(\"data:image/svg+xml,%3C%3Fxml version='1.0' encoding='utf-8'%3F%3E%3C!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0) --%3E%3Csvg version='1.1' id='Layer_1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' fill='${encodeURIComponent(
```

Comment:
Can we remove cruft from there, like `Generator` comment, `id` and `xmlns;*`?

Reply:
If you run it through [SVGOMG](https://jakearchibald.github.io/svgomg/), it will remove a lot of the unnecessary stuff.

Reply:
i would suggest not naming this specific to tooltip, since we will want to use this same beak for any popup or coachmark  that is pointing.